code/src/components/Header.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { useDispatch } from 'react-redux';
+
+import { labyrinth } from '../reducers/labyrinth';
+
+import { OuterFlexWrapper, InnerFlexWrapper } from '../styling/GlobalStyles';
+
+export const Header = () => {
+  const dispatch = useDispatch();
+  const handleRestartButton = () => {
```

Comment:
Do you have this twice? Is it a different button from what you have in the Labyrith.js? 😀